packages/config/src/layer2s/index.test.ts

```diff
@@ -1,10 +1,14 @@
 import { UnixTime } from '@l2beat/shared'
 import { expect } from 'earljs'
 
-import { ProjectTechnologyChoice } from '../common'
+import { KnowledgeNugget, ProjectTechnologyChoice } from '../common'
 import { checkRisk } from '../test/helpers'
 import { layer2s, Layer2Technology, milestonesLayer2s } from './index'
 
+const knowledgeNuggets = layer2s.flatMap<KnowledgeNugget>(
```

Comment:
i am certain that this type here is not needed

Reply:
That's right, I mistook it for reduce function. Removed it